index.html

```diff
@@ -35,9 +35,290 @@ <h3>I am a software developer</h3>
+                    </section>
+
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <section class="tags">
+                                        <ul class="tags1">
+                                            <li>HTML</li>
+                                        </ul>
+                                        <ul class="tags2">
+                                            <li>BOOSTSTRAP</li>
+                                        </ul>
+                                        <ul class="tags3">
+                                            <li>RUBY</li>
+                                        </ul>
+                                    </section>
+                                </section>
+
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <section class="tags">
+                                        <ul class="tags1">
+                                            <li>HTML</li>
+                                        </ul>
+                                        <ul class="tags2">
+                                            <li>BOOSTSTRAP</li>
+                                        </ul>
+                                        <ul class="tags3">
+                                            <li>RUBY</li>
+                                        </ul>
```

Comment:
- [ ] I think the technologies of each project should be a single unordered list. This will make it more semantically correct. I think it should be something like this;
```html
<ul>
  <li>HTML</li>
  <li>BOOTSTRAP</li>
  <li>RUBY</li>
</ul>
```